apps/sms/services/js/message_manager.js

```diff
@@ -191,9 +266,14 @@ var MessageManager = {
     };
   },
 
-  // 0 is a valid value so we need to take care to not consider it as a falsy
-  // value. We want to return null for anything that's not a number or a string
-  // containing a number.
+  /**
+   * Function used to "sanitize" the serviceId. It works as follows:-
```

Comment:
Actually you don't need to use ```Function```, it's clear that it's function :) So you can just use verb in the present tense ```Sanitizes .....```